.codegraph/CODEGRAPH_REPORT.md

```
@@ -1,62 +1,70 @@
 # CodeGraph Report
 
-**Generated:** 2026-06-03T08:20:11.251Z
+Report schema: 1
+Source fingerprint: sha256:c1d44a4509f3611922df3cddfddb56e85f81833b39170388a44599cace03287d
+Tool: codegraph
+Tool version: 1.6.0
+Generation mode: local-index (codegraph status/files)
 
 ## Status
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)